tests/scripts/pgocheck/pefile/ordlookup/__init__.py

```diff
@@ -0,0 +1,31 @@
+from __future__ import absolute_import
+from . import ws2_32
+from . import oleaut32
+
+'''
+A small module for keeping a database of ordinal to symbol
+mappings for DLLs which frequently get linked without symbolic
+infoz.
+'''
+
+ords = {
+    b'ws2_32.dll': ws2_32.ord_names,
```

Comment:
I see.  Presumably this code is the "cost" of using the library for what we want to do with it?  I'm wondering if this might just be simpler to use something like dumpbin /headers instead.  It does seem like we're having to do a lot of work to crack open the PE file when there's already a tool that does this for us.  What do you think?